src/librustc_typeck/check/_match.rs

```diff
@@ -944,13 +950,26 @@ https://doc.rust-lang.org/reference/types.html#trait-objects");
                                            kind_name,
                                            tcx.item_path_str(variant.did),
                                            field_names);
-            if let Some((span, _)) = inexistent_fields.last() {
+            if let Some((span, ident)) = inexistent_fields.last() {
                 err.span_label(*span,
                                format!("{} `{}` does not have {} field{}",
                                        kind_name,
                                        tcx.item_path_str(variant.did),
                                        t,
                                        plural));
+                if plural=="" {
+                    let input = unmentioned_fields.iter().map(|field|
+                                        Symbol::intern(field.to_string().as_str()))
+                                        .collect::<Vec<Symbol>>();
+                    let suggested_name = find_best_match_for_name(input.iter(),
+                                                                &ident.name.as_str(), None);
```

Comment:
Fix indentation above.

Reply:
which IDE or tools I should use to detect this issues?

Reply:
Generally, you can make your changes, run `rustfmt` on the file and only take the changes for the code you changed :-/

That workflow is annoying enough that I just memorized the most relevant rules. Basically, if you have multiple lines

```
like(this,
     right,
     here);
```

you want them to be aligned to the `(`, which is why there's a preference to

```
this_other(
    indentation,
    of,
    multiple,
    args,
);
```

as it is independent of function name length.